.travis.yml

```diff
@@ -3,26 +3,26 @@ cache: packages
 sudo: true
 dist: trusty
 
-addons:
-  apt:
-    packages:
-      - libfreetype6
-
-r:
-- 3.1
-- 3.2
-- oldrel
-- release
-- devel
+# build matrix; turn on vdiffr only on r release
+matrix:
+  include:
+  - r: devel
+  - r: release
+    env: USE_VDIFFR=true
+  - r: oldrel
+  - r: 3.2
+  - r: 3.1
 
 # work around temporary travis + R 3.5 bug
 r_packages: devtools
 
 # Install manually because dev vdiffr is not compatible with Appveyor yet
 r_github_packages: lionel-/vdiffr
 
+# environment variables set for all builds
 env:
   global:
+  # don't treat missing suggested packages as error
   - _R_CHECK_FORCE_SUGGESTS_=false
```

Comment:
It just occurred to me that we could set `USE_VDIFFR` to false here, and then I think the specific matrix env var would override this one.

Reply:
Let me try that.